OpenRA.Game/Settings.cs

```diff
@@ -20,23 +20,61 @@ namespace OpenRA
 {
 	public enum MouseScrollType { Disabled, Standard, Inverted }
 
+	public class LaunchSettings
```

Comment:
Sorry that my initial advice didn't account for these launch-only args, but hacking them into the Settings just so that they get auto-generated man page entries is really not very nice.